Atlas/backend/authentication/tests.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+from django.test import TestCase
+from django.test.client import Client
+
+from jwt_auth import utils
```

Comment:
I'm getting an error regarding this module not being able to imported. Should I do something other than `./manage.py test` inside virtualenv?

Reply:
For the record, as we've talked with @SaitTalhaNisanci, it's `django-jwt-auth == 0.0.2` that should be included in order to run these tests.